varia/style-editor.css

```diff
@@ -1066,51 +1065,85 @@ pre.wp-block-verse {
 	color: currentColor;
 }
 
-.wp-block .has-primary-background-color,
-.has-primary-background-color {
+.has-primary-background-color,
+.has-primary-background-color.has-background-dim {
```

Comment:
I guess this is from another PR where we forgot to rebuild varia...

Reply:
it went away when i rebased...

Reply:
makes sense that it did, I was suspecting it would :)